jsonx/patch.go

```diff
@@ -42,15 +43,20 @@ func isElementAccess(path string) bool {
 	return false
 }
 
+// ApplyJSONPatch applies a JSON patch to an object. It returns an error if the
+// patch is invalid or if the patch includes paths that are denied. denyPaths is
+// a list of path globss (interpreted with [glob.Compile] that are not allowed to
```

Comment:
```suggestion
// a list of path globs (interpreted with [glob.Compile] that are not allowed to
```

Or do you mean the multiple of globs? :D

Reply:
oops